DoSomethingArchive/legacy-website
# Refactoring ds build script for multisite deployments

#### What's this PR do?
- Updates the ds build script to take an '--intl' flag to build on each of the countries' sites  
- Creates a build helper function to abstract the common tasks between single and multisite builds  
- Updates settings.php to use the hostname 'solr' for the solr url instead of an IP address  
- Moves all capistrano tasks to one file and removes extra international includes file
- Adding solr hostname setup to salt for dev environments
- Adding new countries to aliases file
#### Where should the reviewer start?

The most significant changes are in bin/ds in the `build`, `install` and `build_helper` functions.  There are also changes that affect how the arguments for the ds build script are parsed.
#### How should this be manually tested?
- Pull down the code, making sure both sites/dosomething and ../dosomething-mount have the updated intl-deploy code, vagrant destroy your current environment, vagrant up to make sure all Salt changes have happened.
- Run `ds build`, `ds build --intl --install`, `ds pull stage.$AFFILIATE --ds --files`, etc.
#### Any background context you want to provide?

The previous ds build script didn't take a multisite installation into account, this PR addresses that.
#### What are the relevant tickets?

Fixes #DOS-18

cc: @angaither @sergii-tkachenko - please take a look if you're available
